Doc/library/sqlite3.rst

```diff
@@ -1435,13 +1435,24 @@ Cursor objects
       :raises ProgrammingError:
          If *sql* contains more than one SQL statement.
 
+      :raises DeprecationWarning:
```

Comment:
I prefer to only use "raises" when discussing exceptions that will interrupt code execution. The documentation for the warnings module is a little inconsistent on this front — it does sometimes talk about warnings being "raised" — but it seems to more usually discuss warnings being "issued". In other contexts, I've also used the verb "emit" in situations like this.

Reply:
Got it. That makes sense. I'll rework it to be part of the text then. Thanks!

Reply:
Hm, can I omit it from the text, and simply leave it for the `versionchanged` directive?

Reply:
That SGTM!